element_array_ephys/export/nwb/tests.py

```diff
@@ -0,0 +1,37 @@
+from element_array_ephys.export.nwb.nwb import ecephys_session_to_nwb, write_nwb
+
+from pynwb.ecephys import ElectricalSeries
+
+
+def test_convert_to_nwb():
+
+    nwbfile = ecephys_session_to_nwb(
+        dict(subject="subject5", session_datetime="2020-05-12 04:13:07")
```

Comment:
If we're only running tests on subject5, that's only SpikeGLX, and not Open Ephys. It looks like that only makes a difference in terms of which reader we call in [`ecephys_session_to_nwb`](https://github.com/bendichter/element-array-ephys/blob/convert_to_nwb/element_array_ephys/export/nwb/nwb.py#L353-L356), right? There's nothing downstream of that that would motivate another test instance, is there?

Reply:
right